src/commands/rsync.ts

```diff
@@ -20,10 +42,29 @@ export const rsyncCommand = async () => {
         placeHolder: "Which plugin would you like to sync?",
     });
 
-    const wpPath = await vscode.window.showInputBox({
-        prompt: 'Enter the remote path to upload the plugin contents to.',
-        placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
-    });
+	const existingSite = await vscode.window.showQuickPick(getSavedSites(configFilePath), {
+		placeHolder: "Which site would you like to sync to?"
+	});
+
+	if (typeof existingSite === 'undefined') {
+		console.log('No site selected, exiting.');
+		return;
+	}
+
+	let wpPath: string | undefined;
+	if (existingSite === '+ Add New Site') {
+		wpPath = await vscode.window.showInputBox({
+			prompt: 'Enter the remote path to upload the plugin contents to.',
+			placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
+		});
+	} else {
+		wpPath = getSavedPath(existingSite);
+	}
+
+	if (typeof wpPath === 'undefined') {
```

Comment:
Hmm, again, `!wpPath` should do just fine in this case :)

Reply:
49564726207f6fb6f37b26272055c29317fec45f